src/cljc/rems/text.cljc

```diff
@@ -101,8 +105,17 @@
    :application.event/returned :t.applications.events/returned
    :application.event/submitted :t.applications.events/submitted})
 
-(defn localize-event [event-type]
-  (text (get event-types event-type :t.applications.events/unknown)))
+(defn localize-event [event]
+  (text-format
+   (get event-types (:event/type event) :t.applications.events/unknown)
+   (application-util/get-member-name (:event/actor-attributes event))
+   (str/join ", " (mapv application-util/get-member-name
+                        (:event/commenters event)))
+   (str/join ", " (mapv application-util/get-member-name
+                        (:event/deciders event)))
+   (application-util/get-member-name (:application/member event))
+   (str/join ", " (mapv #(localized (:catalogue-item/title %))
+                        (:application/resources event)))))
```

Comment:
This is fine, but means using cryptic `%5` parameters in the localisations. Perhaps having a huge `case` statement base on event types would work better? It would also allow for more custom logic per event.

Reply:
Done